sdk/node/test/accessTokens.js

```diff
@@ -12,141 +12,141 @@ const client = new chain.Client()
 
 let tokenId, clientTokenId, networkTokenId
 
-describe('Access tokens test', () => {
+function createToken() {
+  tokenId = `token-${uuid.v4()}`
```

Comment:
We should try to figure out an alternative to setting a global variable here. Suggestion: wherever you use this function, get the token ID in the `then` clause and set a variable that is local to the `it` example's callback. I think this may eliminate the need for your global variables.